dhctl/pkg/system/ssh/frontend/kube-proxy.go

```diff
@@ -142,7 +142,7 @@ func (k *KubeProxy) tryToRestartFully(startID int) {
 }
 
 func (k *KubeProxy) proxyCMD(startID int) *Command {
-	command := fmt.Sprintf("kubectl proxy --port=%s --kubeconfig /etc/kubernetes/admin.conf", k.port)
+	command := fmt.Sprintf("/opt/deckhouse/bin/kubectl proxy --port=%s --kubeconfig /etc/kubernetes/admin.conf", k.port)
```

Comment:
lets try 

```
command := fmt.Sprintf("PATH=$PATH:/opt/deckhouse/bin/; kubectl proxy --port=%s --kubeconfig /etc/kubernetes/admin.conf", k.port)
```

Reply:
`cmd.Env` does not work, because it will pass for `ssh` command not in command will be executed.
For example, we use same method for post-bootstrap-script  

https://github.com/deckhouse/deckhouse/blob/main/dhctl/pkg/system/ssh/frontend/upload-script.go#L129